src/staticarrays_support.jl

```diff
@@ -0,0 +1,19 @@
+import StaticArrays: StaticArray, tuple_prod
+
+"""
+    StructArrays.staticschema(::Type{<:StaticArray{S, T}}) where {S, T}
+
+The staticschema for a StaticArray is the underlying Tuple used to store the flattened array.
```

Comment:
```suggestion
The `staticschema` of a `StaticArray` element type is the `staticschema` of the underlying `Tuple`.
```

Given that we are referring to an array of arrays, I thought it was important to mention that the `StaticArray` is the element type.